app/models/patient.rb

```diff
@@ -50,7 +50,11 @@ class Patient < ApplicationRecord
 
   has_many :current_prescription_drugs, -> { where(is_deleted: false) }, class_name: 'PrescriptionDrug'
 
+  scope :search_by_address,
+    ->(term) { joins(:address).merge(Address.search_by_street_or_village(term)) }
+
```

Comment:
I know it's throwaway, but can we put this with the other scopes? 😄

Reply:
Ah, yes, this wasn't intentional. Thanks.